src/modules/crud.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-cycle */
+import ul from '../index.js';
```

Comment:
- [ ] [_OPTIONAL_] this is optional but please don't disable linters, if issues come up let's try to fix them.